ggml/src/ggml-cpu/CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ function(ggml_add_cpu_backend_variant_impl tag_name)
                     )
                     if (GGML_MACHINE_SUPPORTS_${tag})
                         set(ARM_MCPU_FLAG_FIX "${ARM_MCPU_FLAG_FIX}+${tag}" PARENT_SCOPE)
-                    else()
+                    elseif(NOT tag STREQUAL "sme")
```

Comment:
Ideally this would be handled by testing if the compiler supports the `+tag` or `-tag`, and skipping adding the flag if it doesn't. Not just for SME, but for all the options.